strength_percentiles.py

```diff
@@ -176,12 +199,189 @@ def populate_database(webpage, database):
                                              # individual's performance at the
                                              # meet
     for row in meet_results_table: # information for one lifter
-        result_and_categories = parse_result_and_categories(row)
+        result_and_categories = parse_row(row)
         if result_and_categories: # row contains results of meet
             cursor.execute(result_storage_string, result_and_categories)
 
     connection.commit()
     connection.close()
 
+####### Pandas functions
+def find_percentile(dataframe, lifts):
+    """
+    Finds the percentile of the input squat, bench, deadlift and total numbers
+    among the sample of results in the database.
+
+    Parameters:
+    -----------
+    dataframe: pandas dataframe
```

Comment:
This could use a more descriptive name. Like population_df, all_lifts_df, or something.